react/src/lib/components/Menu/types/point.ts

```diff
@@ -0,0 +1,4 @@
+export type Point = {
```

Comment:
Thought: Would it be of interest to have Point as a class with related utility functions in utils/geometry.ts as static methods?

Reply:
See above.